bazel/README.md

```diff
@@ -212,7 +219,8 @@ for how to update or override dependencies.
    in your shell for buildifier to work.
 1. `go get -u github.com/bazelbuild/buildtools/buildozer` to install buildozer. You may need to set `BUILDOZER_BIN` to `$GOPATH/bin/buildozer`
    in your shell for buildozer to work.
-1. `bazel build //source/exe:envoy-static` from the Envoy source directory.
+1. `bazel build //source/exe:envoy-static` from the Envoy source directory. For Windows, run
+   `bazel build -c opt //source/exe:envoy-static`.
```

Comment:
I think this is a little clumsy, since it is not a windows issue at all.

It could be worthwhile to document after the original statement at line 215 that one can add -c opt or -c dbg to have an optimized release build, or a fully instrumented and unoptimized debugging build, but this is true on any platform and potentially informative to all builders, not just those working on Windows.

Otherwise LGTM, thanks!

Reply:
Fixed.